apollo-api-impl/pom.xml

```diff
@@ -131,6 +131,20 @@
                 <filtering>true</filtering>
             </resource>
         </resources>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <!-- note: apollo-api-impl splits some packages with apollo-api -->
```

Comment:
So this would really mean thing would have to be moved around? IIRC packages are not supposed to be in multiple modules?

Reply:
Yes eventually. I think for now there's ways around it, either by not depending on two modules that split a package, or keep those dependencies on the classpath.